tag/nitag.yml

```diff
@@ -0,0 +1,1814 @@
+          description: Comma-separated list of case-sensitive keywords
+          type: string
+          required: false
+        - in: query
+          name: properties
+          description: Comma-separated list of name=value strings, e.g. key1=value,key2=value.
+          type: string
+          required: false
+        - in: query
+          name: skip
+          description: Used for pagination. Specifies the number of entries to skip in the response list. Typically used in combination with the take parameter.
+          type: integer
+          required: false
+        - in: query
+          name: take
+          description: Used for pagination. Specfies the number of entries to include in the response list. Typically used in combination with the skip parameter.
+          type: integer
+          required: false
+  /v2/tags-with-values/{path}:
+    get:
+      tags:
+        - tags
+      summary: Get a tag and its current value
+      description: Get a tag and its value
+      operationId: GetTagWithValue
+      parameters:
+        - in: path
+          name: path
+          type: string
+          required: true
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/TagWithValue'
+          description: Success
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}:
+    put:
+      tags:
+        - tags
+      summary: Create or replace a tag
+      description: Replace the tag at the path, or create one if one doesn''t exist. A complete resource representation must be sent (no partial updates).
+      operationId: CreateOrReplaceTag
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+        - in: body
+          name: tag
+          description: Tag to create or replace
+          schema:
+            $ref: '#/definitions/Tag'
+          required: true
+      responses:
+        201:
+          description: 'Created: Tag created successfully'
+        204:
+          description: 'No Content: Tag replaced successfully'
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: 'Conflict: Type change not supported'
+    get:
+      tags:
+        - tags
+      summary: Get a tag
+      description: Get the metadata of a single tag.
+      operationId: GetTag
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            $ref: '#/definitions/Tag'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+    delete:
+      tags:
+        - tags
+      summary: Delete a tag
+      description: Deletes a tag
```

Comment:
This might not affect the description, but just for context: what happens when you delete a tag? Thinking particularly about subscriptions and selections that have already happened.

Reply:
I believe the subscription notifies you about tags that have been deleted or couldn't be found. I think the tag would just disappear from the selection.